packages/big-design/src/components/Button/private.ts

```diff
@@ -1 +1,32 @@
-export { StyleableButton } from './Button';
```

Comment:
Why is this file named `private` instead of `styled` like the others?

Reply:
We are only exporting private internal usage components in this file. Initially, I tried to have the `MessagingButton` in the `styled` file, but module resolution crashed with circular imports.